### ISSUE TITLE
[Accessibility][PDF Viewer] Current result is not announced as a user navigates through search results 

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1747]
**Steps to Reproduce**
1. Enable NVDA screen reader.
2. Open page in Google Chrome.
3. Activate search menu.
4. Input a short text that appears multiple times in the PDF (e.g. "Plan") and activate the search button.
5. Navigate to and activate the "next" result (right arrow icon) button. Observe the visible result number change, but no announcement is made by NVDA.

**Current Behavior**
The user's location in the search results (e.g. "4 of 12") changes visually as the user navigates to previous and next results, but is not announced to screen reader users.

**Expected behavior/code**
When text changes on the page, screen reader users are notified.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.3.1 Info and Relationships

**Possible Solution**
Wrap the search result location (e.g. "4 of 12") with a container that has an aria-live attribute (with a value of "polite") so that the location is announced as users navigate through results.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
